fragments/bastion-ansible.sh

```diff
@@ -138,7 +141,11 @@ function update_etc_hosts() {
     grep -q "$2" /etc/hosts || echo "$1 $2" >> /etc/hosts
 }
 
-update_etc_hosts "$lb_ip" "$lb_hostname.$domainname"
+if [ "$lb_type" == "external" ]; then
+    update_etc_hosts "$master_ip" "$lb_hostname"
```

Comment:
good point, I'll update the patch to update /etc/hosts only on the first run here.